### DISCUSSION TITLE
遇到一个新的同步问题Encountered a new synchronization issue.

### DISCUSSION THREAD

**Discussion by GSanrio** [remotely-save/remotely-save#868] [Q&A]
The following error occurred during synchronization. It was running normally yesterday. Please help me see what kind of error it is and how to solve it.
Working/工作项目/报表改进/需求报表/09出口商品供货（收购）报表.md：The
first argument must be of type string or an
instanceofBuffer,ArrayBuffer,orArrayoran
Array-likeObject.Receivedaninstance of
Object
Smart-chats/UNTITLEDCHAT2024-10-251000
12.md: The first argument must be of type
string oraninstance of Buffer,ArrayBuffer,or
Array or anArray-like Object.Received an
instance ofobject

**Comment by fyears** [comment]
你是用onedrive还是什么呢？

**Reply by GSanrio** [reply]
webdav
